ansible/group_vars/all/all

```diff
@@ -15,6 +15,23 @@ alaska_cloud: alaska
 alaska_homedir: /alaska
 alaska_softiron: 10.4.99.101
 
+# OpenStack fully qualified project name
+project_name: p3@default
```

Comment:
Is this the correct format for the project?

Reply:
It is - since we started using Grafana with support for domains, we need to include the domain.